### PR TITLE
create separate nav2 lib

### PR DESCRIPTION
**Pull request by fred-labs**
Add a `scenario_execution_nav2` library. (actions were previously located in `scenario_execution_ros`.

### CODE REVIEW

**Review by fmirus — APPROVED** [review]

File: libs/scenario_execution_nav2/README.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# Scenario Execution Library for Gazebo
```
Comment (fmirus) [inline libs/scenario_execution_nav2/README.md:1]: copy-paste leftover: Gazebo --> Nav2

File: libs/scenario_execution_nav2/package.xml [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+  <description>Scenario Execution library for Nav2</description>
+  <author email="scenario-execution@intel.com">Intel Labs</author>
+  <maintainer email="scenario-execution@intel.com">Intel Labs</maintainer>
+  <license file="../../LICENSE">Apache-2.0</license>
+
+  <depend>scenario_execution_ros</depend>
+
+  <exec_depend>rclpy</exec_depend>
+  <exec_depend>nav2_simple_commander</exec_depend>
+  <exec_depend>nav2_msgs</exec_depend>
```
Comment (fmirus) [inline libs/scenario_execution_nav2/package.xml:15]: don't we need tf2 and geometry_msgs here as well?

**Review by fmirus — CHANGES_REQUESTED** [review]

**Review by fmirus — APPROVED** [review]

File: libs/scenario_execution_nav2/package.xml (hunk 2 above)
Comment (fred-labs) [reply libs/scenario_execution_nav2/package.xml:15]: fixed

File: libs/scenario_execution_nav2/README.md (hunk 1 above)
Comment (fred-labs) [reply libs/scenario_execution_nav2/README.md]: fixed

**Pull request merged**